devel/git-crypt/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+set realname            git-crypt
+set description_common  {Transparent file encryption in git.}
+
+name                    ${realname}
+homepage                https://www.agwa.name/projects/git-crypt/
+version                 0.6.0
+revision                1
+fetch.type              git
+git.url                 https://github.com/AGWA/${name}.git
```

Comment:
Please use the github PortGroup: https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/github-1.0.tcl.